### PR TITLE
All the formulas from the mpv-player tap

### PR DESCRIPTION
**Pull request by pigoz**
As mentioned in #46413 we are planning to close down [mpv's official tap](https://github.com/mpv-player/homebrew-mpv) when this gets merged. I addressed most of the comments in @mikemcquaid review of the earlier PR (#46413).

The only missing bit fix from the review is renaming the waf resource from `waf-1.8.12` to `waf` (here: https://github.com/Homebrew/homebrew/commit/ec9c416f680ed460730e255fa6a7d2c744afe872#diff-ee2f3e398ed3cc70f2a1bdc443032205R59). @mikemcquaid was suggesting to use a glob but I am not sure how that would work since the resource directory is not the cwd.

How should we handle closing the tap? Do I have to completely delete the repository, or should I empty it out from any formula?

### CODE REVIEW

File: Library/Formula/vapoursynth.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  def python_pkg_config_path
+    Pathname.new(`python3-config --prefix`.chomp)/"lib/pkgconfig"
+  end
+
+  def python_install_package(package)
+    resource(package).stage do
+      system "python3", *Language::Python.setup_install_args(libexec)
+    end
+  end
+end
```
Comment (mikemcquaid) [inline Library/Formula/vapoursynth.rb]: Just hard-code these functions above.

Comment (mikemcquaid) [inline Library/Formula/vapoursynth.rb]: e.g. don't use functions

**Comment by mikemcquaid** [comment]
> The only missing bit fix from the review is renaming the waf resource from waf-1.8.12 to waf (here: ec9c416#diff-ee2f3e398ed3cc70f2a1bdc443032205R59). @mikemcquaid was suggesting to use a glob but I am not sure how that would work since the resource directory is not the cwd.

You could instead use `resource("waf").version` and remove that constant.

File: Library/Formula/vapoursynth.rb (hunk 1 above)
Comment (pigoz) [inline Library/Formula/vapoursynth.rb]: Ok, fixed here and in the `mpv` formula.

**Comment by pigoz** [comment]
> You could instead use resource("waf").version and remove that constant.

Thanks for the pointer! I used that and fixed version parsing for waf URLs since it was broken (8aadfda).

File: Library/Formula/mpv.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  depends_on "libdvdread" => :optional
+  depends_on "libdvdnav" => :optional
+  depends_on "libbluray" => :optional
+  depends_on "libaacs" => :optional
+  depends_on "vapoursynth" => :optional
+  depends_on :x11 => :optional
+
+  depends_on :macos => :mountain_lion
+
+  WAF_VERSION = "waf-1.8.12".freeze
```
Comment (mikemcquaid) [inline Library/Formula/mpv.rb]: Can remove this line now.

**Comment by pigoz** [comment]
Sorry, forgot to remove the `WAF_VERSION` constant. Fixed.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**